modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+[id="modular-documentation-repositories"]
+= Modular Documentation Repositories
+
+IMPORTANT: As CCS strategic imperatives and Pantheon 2 development continue to evolve, the details and recommendations in this section and possibly other sections in this guide might be modified. The Flexible Customer Content (FCC) team recognizes that the fluctuation in recommendations can be frustrating for CCS teams trying to adopt modularization standards and apologizes in advance as the initiative pursues the best possible outcome for CCS documentation.
+
+Due to many factors that determine documentation repository design for individual Red Hat products, the Flexible Customer Content (FCC) initiative cannot provide a strict documentation repository template for all CCS teams to follow. Each team faces different implementation challenges, especially when Red Hat product documentation is single-sourced with corresponding community project documentation.
+
+However, for optimal FCC, Pantheon, and Pantheon 2 compliance, the FCC initiative recommends a general repository structure similar to the following example, where possible:
+
+.Recommended documentation repository structure (where possible)
+[source]
+----
+├── _artifacts
+├── _images
+├── assemblies
+    ├── _artifacts -> ../_artifacts  // Symlinks can be extracted.
+    ├── _images -> ../_images
+    ├── modules -> ../modules
+    ├── assembly-kogito-configuring.adoc
```

Comment:
Let's use the underscore to separate prefixes. It's not a hard requirements, and hyphens are OK, but underscores are more commonly used and more practical from the mod docs perspective. So, for examples, I think we should go with them. (Regardless of what is selected, all examples in the guidelines need to be consistent.)

Reply:
That would mean a very large change to the entire repo, which uses hyphens. We've decided as a team, because there's so much discord over this, that we officially can't really lean to one or the other anymore, and both are viable options. So I'd suggest keeping this in the example, since whichever route we go, we'll likewise have someone coming in to argue for the other.